src/sample/Main.java

```diff
@@ -17,9 +17,9 @@
 public class Main extends Application {
 
     public static void main(String[] args) {
-        BoardGenerator i = new BoardGenerator(30);
-
-        String inputFile = "examples/example.in";
+        BoardGenerator i = new BoardGenerator(7);
+        long start = System.nanoTime();
```

Comment:
Please move it to line 36. We don't need to count time of initialization, reading board etc.